hub/compression.py

```diff
@@ -80,12 +86,14 @@
 
 AUDIO_COMPRESSIONS = ["mp3", "flac", "wav"]
 
+READONLY_COMPRESSIONS = ["mpo", "fli"]
 
 # Just constants
 BYTE_COMPRESSION = "byte"
 IMAGE_COMPRESSION = "image"
 VIDEO_COMPRESSION = "video"
 AUDIO_COMPRESSION = "audio"
+READONLY_COMPRESSION = "read"
```

Comment:
This is not required. Instead a method `is_readonly_compression` would help skip write tests for read only compressions.